ush/python/pygfs/task/stat_analysis.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Dict, List
+from pprint import pformat
+import numpy as np
+from netCDF4 import Dataset
+
+from wxflow import (AttrDict,
+                    FileHandler,
+                    to_fv3time, to_YMD, to_YMDH, to_timedelta, add_to_datetime,
+                    rm_p,
+                    parse_j2yaml, save_as_yaml,
+                    Jinja,
+                    logit,
+                    Executable,
+                    WorkflowException)
+from pygfs.task.analysis import Analysis
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class StatAnalysis(Analysis):
+    """
+    Class for global stat analysis tasks
+    """
+
+    @logit(logger, name="StatAnalysis")
+    def __init__(self, config):
+        super().__init__(config)
+
+        _res = int(self.task_config['CASE'][1:])
+        _window_begin = add_to_datetime(self.task_config.current_cycle, -to_timedelta(f"{self.task_config['assim_freq']}H") / 2)
+        _letkfoi_yaml = os.path.join(self.task_config.DATA, f"{self.task_config.RUN}.t{self.task_config['cyc']:02d}z.letkfoi.yaml")
+
+        # Create a local dictionary that is repeatedly used across this class
+        local_dict = AttrDict(
+            {
+                'APREFIX': f"{self.task_config.RUN}.t{self.task_config.cyc:02d}z.",
+                'jedi_yaml': _letkfoi_yaml
+            }
+        )
+
+        # Extend task_config with local_dict
+        self.task_config = AttrDict(**self.task_config, **local_dict)
+
+    @logit(logger)
+    def initialize(self: Analysis) -> None:
+        """
+        Initialize global stat analysis.
+        """
+        super().initialize()
```

Comment:
Receives the following error:
```
File "/scratch1/NCEPDEV/da/Kevin.Dougherty/global-workflow/ush/python/pygfs/task/analysis.py", line 36, in initialize
    self.task_config.jedi_config = self.get_jedi_config()
File "/scratch1/NCEPDEV/da/Kevin.Dougherty/global-workflow/ush/python/wxflow/logger.py", line 266, in wrapper
    retval = func(*args, **kwargs)
File "/scratch1/NCEPDEV/da/Kevin.Dougherty/global-workflow/ush/python/pygfs/task/analysis.py", line 90, in get_jedi_config
    raise KeyError(f"Task config must contain JCB_BASE_YAML or JEDIYAML")
KeyError: 'Task config must contain JCB_BASE_YAML or JEDIYAML'
```

Reply:
I think you need something like this: https://github.com/CoryMartin-NOAA/global-workflow/blob/49f697a4ac3d32a66d6108ebb3e07a9d9307493e/parm/config/gfs/config.aeroanl#L41